lib/govuk_schemas/version.rb

```diff
@@ -1,4 +1,4 @@
 module GovukSchemas
   # @private
-  VERSION = "4.4.0".freeze
+  VERSION = "4.5.0".freeze
```

Comment:
I wonder if this is more suited to being a patch release rather than a minor release. It seems like this is a bug-fix rather than a new feature. 🤔

```suggestion
  VERSION = "4.4.1".freeze
```

What do you think?

Reply:
yea good shout. Will update 👍